pipeline/unified/channel/sign-release-package-windows.yaml

```diff
@@ -36,11 +75,11 @@ jobs:
                 TargetFolder: '$(System.DefaultWorkingDirectory)/signing-in-progress/${{ parameters.signedArtifactName }}'
 
           - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
-            displayName: 'Send to ESRP Code Signing service'
```

Comment:
Since we call ESRP twice, I updated the display names to better describe which step we're on